azure-core/azure/core/async_paging.py

```diff
@@ -28,9 +28,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+#pylint: disable=too-few-public-methods
 class AsyncPagedMixin(AsyncIterator):
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, **kwargs): #pylint: disable=unused-argument
```

Comment:
I didn't notice this when copying paging over, but this warning seems indicative of a real issue, namely that there ought to be a call to `super` in this initializer to soak up `args` and `kwargs` and make sure that they reach parent or sibling classes. Otherwise using this Mixin seems very brittle as what initializers get what args will depend on the order the base classes in the class definition.  Thoughts @annatisch?

Reply:
getting the args and kwargs from super.